src/app.js

```diff
@@ -1,9 +1,16 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import routes from './routes';
+import database from './config/database'
 
 const app = express();
-app.use(bodyParser.json());
-app.use('/', routes);
 
-export default app;
+export default () => {
+  return database.connect()
+  .then(() => {
```

Comment:
I usually try to avoid the usage of anonymous functions with promises, I normally create a separated function like:

```js
database.connect()
  .then(configureExpress);
```